### ISSUE TITLE
Swipe to nth page smoothly

### ISSUE THREAD

**Issue by brainahdev** [react-native-community/react-native-tab-view#343]
the tabview sample seems awesome and is that can we able to transit nth page? I didnt get how to do that. Is there any sample?

**Comment by harikanammi** [comment]
I am using  react-native-scrollable-tab-view. In this tab-view it swipes to nth page.
I am using this link https://github.com/skv-headless/react-native-scrollable-tab-view.

Here is my Code:
```

import React,{Component} from 'react';
import {
  Dimensions,
  StyleSheet,
  ScrollView,
  View,
  Alert,
  AlertIOS,
  AsyncStorage,
  ActivityIndicator,
  Image,
  Text,
  TouchableOpacity,
} from 'react-native';

import ScrollableTabView, {ScrollableTabBar,DefaultTabBar } from 'react-native-scrollable-tab-view'


class ScrollableTabView extends Component {
  constructor(props){
      super(props);
      this.state={
      }

    }
    render(){
      return(
        <View style ={{flex:1}}>

        <View style={styles.tabBarView}>
          <ScrollableTabView
            style={{ }}
              initialPage={2}
            renderTabBar={() => <ScrollableTabBar />}>

            <View tabLabel='All' style={{flex:1, alignItems:'center', justifyContent:'center'}}>
          <Text style={{color:'black', fontSize:15}}>This is all types</Text>
            </View>

            <View tabLabel='Tea' style={{flex:1, alignItems:'center', justifyContent:'center'}}>
            <Text style={{color:'black', fontSize:15}}>This is Tea types</Text>
            </View>
            <View tabLabel='Coffee' style={{flex:1, alignItems:'center', justifyContent:'center'}}>
            <Text style={{color:'black', fontSize:15}}>This is Coffee types</Text>
            </View>
            <View tabLabel='Food' style={{flex:1, alignItems:'center', justifyContent:'center'}}>
            <Text style={{color:'black', fontSize:15}}>This is Food types</Text>
            </View>
            <View tabLabel='All' style={{flex:1, alignItems:'center', justifyContent:'center'}}>
          <Text style={{color:'black', fontSize:15}}>This is all types</Text>
            </View>
            <View tabLabel='Tea' style={{flex:1, alignItems:'center', justifyContent:'center'}}>
            <Text style={{color:'black', fontSize:15}}>This is Tea types</Text>
            </View>
            <View tabLabel='Coffee' style={{flex:1, alignItems:'center', justifyContent:'center'}}>
            <Text style={{color:'black', fontSize:15}}>This is Coffee types</Text>
            </View>
            <View tabLabel='Food' style={{flex:1, alignItems:'center', justifyContent:'center'}}>
            <Text style={{color:'black', fontSize:15}}>This is Food types</Text>
            </View>
            <View tabLabel='All' style={{flex:1, alignItems:'center', justifyContent:'center'}}>
          <Text style={{color:'black', fontSize:15}}>This is all types</Text>
            </View>
            <View tabLabel='Tea' style={{flex:1, alignItems:'center', justifyContent:'center'}}>
            <Text style={{color:'black', fontSize:15}}>This is Tea types</Text>
            </View>
            <View tabLabel='Coffee' style={{flex:1, alignItems:'center', justifyContent:'center'}}>
            <Text style={{color:'black', fontSize:15}}>This is Coffee types</Text>
            </View>
            <View tabLabel='Food' style={{flex:1, alignItems:'center', justifyContent:'center'}}>
            <Text style={{color:'black', fontSize:15}}>This is Food types</Text>
            </View>
        </ScrollableTabView>

        </View>


        </View>


        );
    }


  }

  module.exports = ScrollableTabView;
```

**Comment by satya164** [comment]
Hey, I just released a new alpha `2.0.0-alpha.0` of the library. It's rewritten using [react-native-gesture-handler](https://github.com/kmagiera/react-native-gesture-handler) and [react-native-reanimated](https://github.com/kmagiera/react-native-reanimated) addresses a many platform specific bugs and performance problems. The documentation is updated as well.

Please try the new version and see if it addresses your issue. If not, please open a new issue following the issue template.
